share/availability/discovery/discovery.go

```diff
@@ -90,15 +58,25 @@ type OnUpdatedPeers func(peerID peer.ID, isAdded bool)
 func NewDiscovery(
 	h host.Host,
 	d discovery.Discovery,
-	params Parameters,
+	opts ...Option,
 ) *Discovery {
+	params := DefaultParameters()
+	// TODO: DefaultParameters was changed to not include `discoveryRetryTimeout`
+	// since it's a private field and it breaks nodebuilder/config_test.go
+	// so this is gonna be where we set the default value for it for now.
+	params.discoveryRetryTimeout = defaultRetryTimeout
```

Comment:
Ah, that's annoying but makes sense

Reply:
I agree

Reply:
This made its way to discovery through the revamp PR, so dunno what's the rational honestly, but it makes sense to turn it into just a var